operations/workflow.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+###########################################################
+# Description: Simplifies git workflow on remote machines #
+#                                                         #
+# Usage: ./workflow.sh -opt                               #
+#                                                         #
+# Options: -b or -branch to clone a branch                #
+#          -c or -clean to delete a branch                #
+###########################################################
+
+BRANCH_PATH="$HOME/branch/"
+REPO="https://github.com/bcgov/GDX-Analytics-microservice.git"
+
+while getopts ":b(branch) :c(clean)" opt; do
+  case $opt in
+    b)
+      read -p "Enter the branch label: " branch
+      git ls-remote --heads ${REPO} ${branch} | grep ${branch} >/dev/null
+      if [ "$?" == "1" ] # Check exit code of git ls-remote
+      then
+              echo ""$branch" doesn't exist in "$REPO""
+              exit
+      fi
+      read -p "Enter your GitHub Name (First name Last name): " username
```

Comment:
Can this just read "Enter your GitHub username"?

Reply:
I see that my variable name is confusing. Git config user.name is actually First and Last name, not username.
https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup/#_your_identity

Reply:
More direct - https://docs.github.com/en/github/getting-started-with-github/getting-started-with-git/setting-your-username-in-git

Reply:
then it's not a "GitHub name"; let's just change it to "Enter your first and last name:" avoiding the confusion coming from multiple meanings of username.

Reply:
I've updated the prompt to be more straight forward